docs/DeveloperGuide.md

```diff
@@ -140,13 +140,13 @@ The `Model` component,
 <img src="images/StorageClassDiagram.png" width="550" />
```

Comment:
I think you could make the width bigger so that the text size in the diagram matches the text size of the main text of the diagram.